<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #495057;</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>451</width>
     <height>421</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #8d99ae;</string>
   </property>
   <widget class="QLabel" name="FaceArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>431</width>
      <height>401</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>D:/Coding/Project/#Project Designs Equipment/Image/15437639.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>450</y>
     <width>311</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #8d99ae;</string>
   </property>
   <widget class="QLabel" name="PredictedLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>291</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(162, 162, 162);
font: italic 14pt &quot;Monotype Corsiva&quot;;</string>
    </property>
    <property name="text">
     <string>Capture Your Picture For Finding Your Expression </string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="BTNCaptureFacial">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>141</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #edf2f4;
font: 75 italic 15pt &quot;Book Antiqua&quot;;
color:rgb(113, 113, 113)</string>
   </property>
   <property name="text">
    <string>Capture</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
